app/plugins/logger.js

```diff
@@ -36,6 +42,8 @@ module.exports = {
     formatters: {
       level: (level) => ({ level })
     },
+    ignorePaths: ['/healthy', '/healthz'],
+    ignoreTags: ['assets'],
```

Comment:
Do we have another way of knowing if these are failing?

Reply:
Yes application insights picks up failed requests, and we have alerts based on those too looking for non success response codes. The logging of these things doesn;t give us a lot except noise